index.html

```diff
@@ -48,4 +48,178 @@ <h2 class="hex connect">LET’S CONNECT</h2>
+            <li class="o-li">Back End Dev</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
+            </div>
+            </div>
+            </section>
+            <section class="grid-items other-projects">
+            <img class="project-img" src="images/projects/project-3.png" alt="Project-1" />
+            <div class="details">
+            <div class="titles">
+            <h3 class="project-title">Tonic</h3>
+            <ul class="canopy">
+            <li>CANOPY</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">Back End Dev</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
+            </div>
+            </div>
+            </section>
+            <section class="grid-items other-projects">
+            <img class="project-img" src="images/projects/project-4.png" alt="Project-1" />
+            <div class="details">
+            <div class="titles">
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <ul class="canopy">
+            <li>CANOPY</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">Back End Dev</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
+            </div>
+            </div>
+            </section>
+            </div>
+            <div class="about-myself">
+            <h2 class="about">About Myself</h2>
+            <p class="text">
```

Comment:
- [ ] Please make sure the `about section` is` section` tag instead of `div `tag. As mentioned above using the `section` element over `div` makes your code more accessible